styles/lib/carnival/_theme.scss

```diff
@@ -30,3 +30,7 @@ $Grey1: #EEEAE0;
 $Grey2: #757575;
 $black: #000000;
 $white: #FFFFFF;
+
+// Fonts
+$titleOption1Font: Mulish, sans-serif;
```

Comment:
what would you think of using `$mulish` as the variable name here? fonts/colors were originally named vaguely so that the value of `titleOption1` font could be changed without changing the name ... but I don't think we use that ability much. to me, we get more value from clear variable names. what do you think?